tasks/gpg_keys.yml

```diff
@@ -1,12 +1,25 @@
 ---
+- name: Ensure GPG keys have full fingerprints.
+  assert:
+    that:
+      - "'fingerprint' in item.keys()"
+      - item.fingerprint|length == 40
+    fail_msg: >-
+      GPG public key fingerprints music be long-form 40 characters
+  with_items: "{{ grsecurity_build_gpg_keys }}"
+
 - name: Import GPG keys for building Linux kernel.
-  command: gpg --keyserver {{ grsecurity_build_gpg_keyserver }} --recv-keys "{{ item.fingerprint }}"
+  shell: gpg --import - <<< "{{ lookup('file', item.fingerprint+'.pub')|escape }}"
```

Comment:
We've also stored GPG keys in variables and base64 encoded them out and back in -- that may make more sense if you want end-users to be able to over-ride keys via ansible vars. Please don't change it though -- just sort of talking out my ass here to pretend i gave this a thorough review ;)

By the way, I like your use of naming the public keys by fingerprint 👍